spec/models/institution_builder_spec.rb

```diff
@@ -462,7 +462,7 @@
       let(:ipeds_ic_py) { IpedsIcPy.first }
       let(:ipeds_ic_ay) { IpedsIcAy.first }
 
-      let(:nil_ipeds_ic_ay) { IpedsIcPy::COLS_USED_IN_INSTITUTION.each_with_object({}) { |v, o| o[v] = nil } }
+      let(:nil_ipeds_ic_ay) { IpedsIcPy::COLS_USED_IN_INSTITUTION.index_with({}) { |v, o| o[v] = nil } }
```

Comment:
Can this be moved into a separate PR

Reply:
It won't pass continuous-integration/jenkins/branch without it